assets/templates/partials/census/get-data.tmpl

```diff
@@ -0,0 +1,34 @@
+{{ $length := len .Version.Downloads }}
+<section id="get-data" class="u-mt-l u-mb-l" aria-label="{{ localise "GetData" .Language 1 }}">
+    <h2 class="u-fw-b u-mt-l u-pb-no u-pt-no">{{ localise "GetData" .Language 1 }}</h2>
+    <p class="u-fs-r u-mt-s u-mb-s u-pt-no u-pb-s">{{ localise "GetDataLeadText" .Language 1 }}</p>
+    <ul class="list list--bare list--icons u-fs-r u-pb-s">
+        {{ range $i, $el := .Version.Downloads }}
+            <li class="list__item u-mt-s u-mb-no u-pl-no u-pb-s u-pt-no u-fs-r{{if notLastItem $length $i }} u-bb{{end}}">
+                <span class="list__prefix">
+                    <svg
+                        class="svg-icon"
+                        width="14"
+                        height="17"
+                        viewBox="0 0 14 17"
+                        fill="none"
+                        xmlns="http://www.w3.org/2000/svg"
+                        focusable="false">
+                        <path d="M0 17H14V15H0V17ZM14 6H10V0H4V6H0L7 13L14 6Z" fill="#222222"/>
+                    </svg>
+                </span>
+                <span class="u-fs-r u-m-no u-p-no">
+                    <a href="{{ .URI }}" class="list__link underline-link">{{ .Name }}</a>
```

Comment:
The [download resources pattern](https://ons-design-system.netlify.app/patterns/download-resources/) in the design system showcases the use of a visually hidden element that contains the download meta information; I suspect this is so that screen reader users can get that information immediately when landing on the link

The `download__meta` element then has `aria-hidden=true` to remove the potential for screen reader users to hear that message twice. I think we should update the markup to align more with this pattern

Example here:
```
<h3 class="ons-u-fs-m ons-u-mt-no ons-u-mb-xs">
    <a href="#link">
        Census awareness poster
        <span class="ons-u-vh">, PDF document download, 499KB, 1 page</span>
    </a>
</h3>
<span class="ons-u-fs-s ons-u-mb-xs ons-download__meta" aria-hidden="true">
     Poster, PDF, 499KB, 1 page
</span>
```

Let me know if you think otherwise

Reply:
The pattern is experimental and does require further research/evidence to support the markup used. 

However, I do agree that a screen user reader could benefit from additional information when they land on the link. What that should be is ambiguous. Currently, the mac screen reader will announce the file name and the format therefore the additional information needed would be the file size, and possibly the 'supporting information' tag. In addition, the mac screen reader can handle the link titles with hyphens reasonably well but what about the others? That being said, the rendered markup could be something like:

``` html
<a href="https://downloads.com/file.xls" class="list__link underline-link">ethnicity-data.xls
     <span class="u-vh"> download, including supporting information, 123KB</span>
</a>
```
... and the associated `download__meta` 
``` html
<span class="u-fs-s u-mb-xs u-mt-xs download__meta" aria-hidden="true"> 
          Includes supporting information, XLS (76.8 KB)
</span>
```
What do you think?

Reply:
`aria-hidden="true"` causes display conflicts between Sixteens and the design system and there's no utility class to override `visibility: hidden`; the aria tag will not be included. I have made a note on the backlog ticket to include this when the modularised design system is adopted.